src/gemm/kernels/x86_64.rs

```diff
@@ -18,9 +21,11 @@ impl Kernel for FmaKernel {
     }
 
     fn supported() -> bool {
-        is_x86_feature_detected!("fma")
+        is_x86_feature_detected!("avx2") && is_x86_feature_detected!("fma")
```

Comment:
This fixes an inconsistency between the GEMM kernel's `target_feature`s and this method. I'm not certain whether there are CPUs for which `fma` is detected but `avx2` is not, but there should definitely be consistency between this function and the kernel.